benchmarks/markdown/page-template.js

```diff
@@ -0,0 +1,25 @@
+const faker = require(`faker`)
+const matter = require(`gray-matter`)
+
+const NUM_ROWS = process.env.NUM_ROWS || 25;
```

Comment:
Note: swapping this to even 100 or so fails the build with OOM.

e.g.

``bash
NUM_ROWS=100 yarn build
```

will reliably fail, so this is a good, general-purpose test case/template.